### ISSUE TITLE
[Popup] Compatibility issue with existing `<popup>` use

### ISSUE THREAD

**Issue by tomayac** [openui/open-ui#341]
The UA stylesheet changes introduced in [crbug.com/1168738#c6](https://bugs.chromium.org/p/chromium/issues/detail?id=1168738#c6) seem to cause [compatibility issues](https://support.google.com/chrome/thread/106244569) with prior use of `<popup>`. While this is self-inflicted and was predicted (see @annevk's [comment](https://github.com/WICG/webcomponents/issues/658#issuecomment-366508364) [I know this was written in the context of CE and the present issue is not about a CE]), it may seem nevertheless worthwhile to explore how much of a compatibility issue this is, especially since the Intent to Prototype message mentioned there were [no compatibility issues](https://groups.google.com/a/chromium.org/g/blink-dev/c/9y-Thg9UCxY/m/_4gShWjQAAAJ#:~:text=Interoperability%20and%20Compatibility-,None,-Gecko%3A%20No%20signal).

**Comment by pepelsbey** [comment]
Some real life examples:

![photo_2021-04-22 12 30 53](https://user-images.githubusercontent.com/105274/115699372-44be8a00-a36e-11eb-82ed-99fb266a82a6.jpeg)

Reported by developers who (despite of [Angular’s recommendations](https://angular.io/guide/styleguide#component-custom-prefix) to prefix components) used `<popup>` element as it.

**Comment by gregwhitworth** [comment]
Interesting - @mfreed7 @melanierichards either of you want to do some compat analysis?

**Comment by melanierichards** [comment]
Saw this on Twitter while out of office (thanks again folks for filing)...maybe we should hold on a compat analysis until we have a better idea on names for this element are reasonably in scope. Un-prefixed components might pose a problem with any name that is natural to reach for; it'll likely come up again with other new elements; and I don't think we should hold back the platform for [improperly named custom elements](https://html.spec.whatwg.org/multipage/custom-elements.html#valid-custom-element-name). This issue is precisely why the author requirement for a hyphen exists. :) But if there's 2-3 names that seem reasonable and one of these minimizes conflicts with un-prefixed custom elements, maybe that's our winner.

Also, for awareness: https://bugs.chromium.org/p/chromium/issues/detail?id=1168738#c28

**Comment by mfreed7** [comment]
This is an interesting issue. I agree that I completely missed this potential issue in my intent to prototype. It seems that the compat concerns aren't *awful*, as there have only been a handful of [bugs](https://crbug.com/1199964) reported, and most were quickly resolved on the site-side by renaming the element. But it would definitely be good to do this analysis in advance of actually shipping either `<popup>` or `<selectmenu>` to the web platform. 

@melanierichards thanks for linking that commit. That was in response to this bug: https://crbug.com/1199964. When I added `<popup>` to the codebase behind a flag, I didn't prevent the UA stylesheet from doing a `display:none !important` for closed popups, which hid any non-standard `<popup>` elements a site used. This has been fixed now in Chromium M91, but not in (already stable) M90.
